crates/python/src/qpu/translation.rs

```diff
@@ -88,33 +90,59 @@ py_wrap_error!(
     PyRuntimeError
 );
 
+#[derive(Copy, Clone)]
+pub enum TranslationBackend {
+    V1,
+    V2,
+}
+
+py_wrap_simple_enum! {
+    PyTranslationBackend(TranslationBackend) as "TranslationBackend" {
+        V1,
+        V2
+    }
+}
+
 #[derive(Clone, Default)]
 #[pyclass(name = "TranslationOptions")]
-pub struct PyTranslationOptions(TranslationOptions);
+pub struct PyTranslationOptions {
+    inner: TranslationOptions,
+    backend: Option<TranslationBackend>,
+}
 
 impl PyTranslationOptions {
     pub fn as_inner(&self) -> &TranslationOptions {
-        &self.0
+        &self.inner
     }
 }
 
 #[pymethods]
 impl PyTranslationOptions {
     #[new]
     fn __new__() -> PyResult<Self> {
-        Ok(Self(Default::default()))
+        Ok(Self {
+            inner: Default::default(),
+            backend: None,
+        })
+    }
+
+    #[getter]
+    fn backend(&self) -> Option<PyTranslationBackend> {
+        self.backend.map(Into::into)
     }
 
     fn use_backend_v1(&mut self) {
-        self.0.use_backend_v1()
```

Comment:
Isn't this method already available in Python?

Reply:
Not sure what you mean?